appConfig.js

```diff
@@ -80,25 +80,23 @@ const config = {
     'EQIDAQAB\n' +
     '-----END PUBLIC KEY-----',
   heroData: {
-    staffPicks: {
+    'staffPicks': {
       category: '',
       header: 'Staff Picks',
       description: 'This is staff picks page.',
       heroImageUrl: 'src/client/images/desktop.teens.cover.2017.png',
     },
-    annual: {
-      ya: {
-        category: '',
-        header: 'Best Books for Teens',
-        description: 'Explore our annual selection of outstanding young adult titles.',
-        heroImageUrl: 'src/client/images/desktop.teens.cover.2017.png',
-      },
-      childrens: {
-        category: '',
-        header: 'Best Books for Kids',
-        description: 'Explore our annual selection of outstanding children\'s titles.',
-        heroImageUrl: 'src/client/images/desktop.kids.cover.2017.png',
-      },
+    ya: {
```

Comment:
If `staffPicks` is getting single quotes, is there any reason why `ya` and `childrens` are not getting single quotes?

Reply:
ohh it should not be there! Thanks

Reply:
And updated!